doc-jsdoc/pentaho/config/IService.jsdoc

```diff
@@ -15,33 +15,33 @@
  */
 
 /**
- * The `IConfigurationService` interface describes a service
- * that manages configurations of [value types]{@link pentaho.type.Value.Type}
- * of the Pentaho Client Metadata Model.
+ * The `config.IService` interface describes a service
+ * that manages configurations, usually of types.
  *
- * The [add]{@link pentaho.type.IConfigurationService#add} method
- * is used to register a [spec.ITypeConfiguration]{@link pentaho.type.spec.ITypeConfiguration}
+ * The platform's main configuration service instance can be obtained by requiring
+ * the module `pentaho/service!pentaho.config.IService?single`.
+ *
+ * The platform registers an instance module that
+ * loads all registered configuration rule set modules
+ * (i.e., modules of type {@link pentaho.config.spec.IRuleSet}).
+ *
+ * The [add]{@link pentaho.config.IService#add} method
+ * is used to register a [config.spec.IRuleSet]{@link pentaho.config.spec.IRuleSet}
  * to the service.
  *
- * The [select]{@link pentaho.type.IConfigurationService#select} method
- * selects the rules of added type configurations that apply to
+ * The [select]{@link pentaho.config.IService#select} method
```

Comment:
I still prefer _extracting_ more information from the class name itself, without knowing the package. I mean `IConfigurationService`. But is just a matter of personal taste, not a blocker.